### PR TITLE
Fixing Examples

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Removing brackets from a few examples as they tend to cause confusion.
Also updated some placeholder text to match our [style guide for examples](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#placeholder-text)

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/core/connect-data-platform/databricks-setup.md [hunk 1]
```diff
@@ -76,14 +76,14 @@ your_profile_name:
...
+      http_path: /SQL/YOUR/HTTP/PATH # Required
       auth_type: oauth # Required if using OAuth-based authentication
-      client_id: [OAuth-Client-ID] # The ID of your OAuth application. Required if using OAuth-based authentication
-      client_secret: [XXXXXXXXXXXXXXXXXXXXXXXXXXX] # OAuth client secret. # Required if using OAuth-based authentication
-      threads: [1 or more]  # Optional, default 1
+      client_id: OAUTH_CLIENT_ID # The ID of your OAuth application. Required if using OAuth-based authentication
+      client_secret: XXXXXXXXXXXXXXXXXXXXXXXXXXX # OAuth client secret. # Required if using OAuth-based authentication
+      threads: 1_OR_MORE  # Optional, default 1
 ```
 </File>
```
Comment (runleonarun) [inline website/docs/docs/core/connect-data-platform/databricks-setup.md:88]: I think we should change the options that show up in the tables below. Like [here](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/databricks-setup?tokenoauth=oauth#host-parameters) entries in the [authentication parameters table](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/databricks-setup?tokenoauth=oauth#authentication-parameters) might be confusing especially now they don't match the example.

File: website/docs/docs/core/connect-data-platform/hive-setup.md [hunk 2]
```diff
@@ -72,15 +72,15 @@ your_profile_name:
   outputs:
     dev:
```
Comment (runleonarun) [inline website/docs/docs/core/connect-data-platform/hive-setup.md:73]: There are more examples higher up that need replacing. Like the [Insecure section](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/hive-setup#insecure). And the example [here](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/hive-setup#connection-methods)

**Review by runleonarun — CHANGES_REQUESTED** [review]
Hey @matthewshaver I know you wanted to get the crucial examples, but it might be more confusing if we don't change all examples at least on a page. 

I suggest we clean up these pages completely then chip away at other config and setup pages. What do you think?

File: website/docs/docs/core/connect-data-platform/hive-setup.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/docs/core/connect-data-platform/hive-setup.md:73]: Not sure how I missed those. Thank you!

File: website/docs/docs/core/connect-data-platform/databricks-setup.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/docs/core/connect-data-platform/databricks-setup.md:88]: Good call!

File: website/docs/docs/core/connect-data-platform/hive-setup.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ dbt-hive supports two transport mechanisms:
 - binary
 - HTTP(S)
 
-The default mechanism is `binary`. To use HTTP transport, use the boolean option `use_http_transport: [true / false]`.
+The default mechanism is `binary`. To use HTTP transport, use the boolean option. For example, `use_http_transport: true`.
```
Comment (runleonarun) [inline website/docs/docs/core/connect-data-platform/hive-setup.md:33]: @matthewshaver I changed this to be an example for clarity.

File: website/docs/reference/resource-configs/databricks-configs.md [hunk 4]
```diff
@@ -442,9 +442,9 @@ To configure this inside of dbt Cloud, use the [extended attributes feature](/do
 
 compute:
   Compute1:
-    http_path:[`/some/other/path']
+    http_path: /SOME/OTHER/PATH
   Compute2:
-    http_path:[`/some/other/path']
+    http_path: /SOME/OTHER/PATH
 
 ```
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/databricks-configs.md:467]: I noticed the profile.yml Under [Selecting compute per model](https://docs-getdbt-bwqcghv0s-dbt-labs.vercel.app/reference/resource-configs/databricks-configs#selecting-compute-per-model) also uses the `<profile-name>` but maybe we should change all profile.yml and project.yml examples in one PR to reduce confusion. what do you think?

**Review by runleonarun — APPROVED** [review]
@matthewshaver approved with a question for ya!

**Pull request merged**